helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -50,6 +50,8 @@ splunkPlatform:
   disableCompression: true
   # HTTP timeout when sending data. Defaults to 10s.
   timeout: 10s
+  # Idle connection timeout. defaults to 10s
+  idleConnTimeout: 10s
```

Comment:
Is the current default 10s? It seems a little aggressive especially if using connection pooling

Reply:
the default will be 10s going forward. 30s is the old default. The value has changed based on the defaults and best practices of Splunk Enterprise (set by `busyKeepAliveIdleTimeout` and defaults to 12 seconds).
PR on contrib: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/20653
Discussion: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/20543